### PR TITLE
Alex progress

### PR DESCRIPTION
**Pull request by alexling92**

### CODE REVIEW

File: virtual-cluster/alex.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+> - Download the Proxmox Repository key: `wget http://download.proxmox.com/debian/proxmox-ve-release-5.x.gpg -O /etc/apt/trusted.gpg.d/proxmox-ve-release-5.x.gpg`
+> - Update repository and system: `apt update && apt dist-upgrade`
+> - Now that all the preparation has been done, proceed to install Proxmox: `apt-get install proxmox-ve`
+> - Go to GCP --> VPC network --> firewall rules and create a new firewall rule for tcp:8006. This is to allow access to Proxmox VE web interface.
+> - Done! Access the Proxmox admin web interface by typing https://youripaddress:8006 in the web browser.  
+>
+>Upon reaching this point, a Debian nested VM hosting Proxmox VE would be created.
+***
+### *Creating a cluster and adding all nodes*
+> Created 2 more VM instances based on the previous step to serve as nodes for the VM cluster. Tried to set up a cluster using the Proxmox web interface but was faced with a lot of issues during the process of adding the 2nd node to the cluster.
```
Comment (eugene-chow) [inline virtual-cluster/alex.md:30]: I believe adding a node to the cluster can only be done on the command line

**Review by eugene-chow — APPROVED** [review]

**Comment by eugene-chow** [comment]
@CloudCommandos/infra Remember to `Merge pull request` after the PR is approved

**Comment by eugene-chow** [comment]
Good work. Thanks!

File: virtual-cluster/alex.md (hunk 1 above)
Comment (alexling92) [reply virtual-cluster/alex.md:30]: I am able to add a node in to the cluster using the proxmox web interface function. However some configurations need to be done as GCP doesnt support multicast.

**Comment by eugene-chow** [comment]
Good work. Please document it.

**Pull request merged**